error-handling/erros.js

```diff
@@ -1,4 +1,20 @@
-exports.handle500s = (err, req, res, next) => {
+exports.handle404s = (err, req, res, next) => {
   console.log(err);
-  res.status(500).send({ msg: "Internal Server Error" });
+  res.status(404).send({ msg: "Not Found" });
+};
+
+exports.handleCustomErros = (err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(404).send({ msg: err.msg });
+  } else {
+    next(err);
+  }
+};
+
+exports.handleSQLErrors = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Invalid id input" });
+  } else {
+    next(err);
+  }
```

Comment:
Please add a 500 error handler into your app - this should be the final error handler, where everything else ends up if it doesn't get caught by the other bits